config/500-controller.yaml

```diff
@@ -63,10 +63,10 @@ spec:
           value: knative.dev/sources
         - name: GL_RA_IMAGE
           value: ko://knative.dev/eventing-gitlab/cmd/receive_adapter
-        - name: POD_NAMESPACE
```

Comment:
Sorry, I am a bit out of context here.

I know we now inject `POD_NAME` to controllers, but don't we need `POD_NAMESPACE` anymore?

Reply:
I don't know about any controller using `POD_NAMESPACE` in Knative, and this one is no exception. Not sure where that came from.

Reply:
LOL, thanks for the explanation.

/lgtm
/approve